docs/conf/australia/2021/sponsors/prospectus.rst

```diff
@@ -0,0 +1,282 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/{{ shortcode }}-group.png
+
+Online Sponsorship Prospectus
```

Comment:
Do we need to update these prices?

Reply:
Will have a chat with @ericholscher before we go full launch.